main.cpp

```diff
@@ -127,184 +171,172 @@ struct CarWash
  if your code produces a -Wpadded warning, add '-Wno-padded' to the .replit file with the other compiler flags (-Weverything -Wno-missing-prototypes etc etc)
  */
 
-
-/* Thing 1) Audio Interface
-5 properties:
-    1) number of ADAT inputs (int)
-    2) number of XLR inputs (int)
-    3) number of headphone outputs (int)
-    4) percentage of THD at 1kHz of the line outputs (float)
-    5) amount of watts required to operate (float)
-3 things it can do:
-    1) link analogue channel 1 and 2 together
-    2) adjust the playback level of main
-    3) mute the main monitors
-*/
 #include <iostream>
 #include <string>
 
 struct AudioInterface
 {
-    // number of ADAT inputs
     int numOfAdatInput = 2;
-    // number of XLR inputs
     int numOfXlrInputs = 4;
-    // number of headphone outputs
     int numOfHeadphonesOutputs = 6;
-    // percentage of THD at 1kHz of the line outputs
     float percentageOfTHD = 0.0001f;
-    // amount of watts required to operate
     float amountOfWattsRequired = 29.5f;
+    float levelControl = 10.f;
 
-    // link analogue channel 1 and 2 together
     void linkAnalogueChannels1And2();
-    // adjust the playback level of main
-    void mainsPlaybackLevel(int displayLevel);
-    // mute the main monitors
-    void muteMainMonitors(bool pressMuteButton);
+    float mainsPlaybackLevel(float displayLevel);
+    void muteMainMonitors();
 };
 
-/*
-Thing 2) cafe
-5 properties:
-    1) number of staff working (int)
-    2) number of tables used (int)
-    3) amount of food wastage in KG (double)
-    4) soup of the day (std::string)
-    5) average cost of a meal in dollars (double)
-3 things it can do:
-    1) serve speciality drinks
-    2) provide table service
-    3) serve vegan food
- */
+    void AudioInterface::linkAnalogueChannels1And2()
+    {
+    }
+    float AudioInterface::mainsPlaybackLevel(float displayLevel)
+    {
+        displayLevel = amountOfWattsRequired * levelControl;
+        return displayLevel;
+    }
+    void AudioInterface::muteMainMonitors()
+    {
+    }
```

Comment:
Align this code with the left edge of the text editor
There is no reason for it to be indented
- [ ] revise